designs/containers/vnf-deployment/vnf-deployment.md

```diff
@@ -1,44 +1,41 @@
-
 
 ### Nuage SDN for Kubernetes
-Nuage does not present a restriction with respect to where the container VNF should be deployed. The VNF can be deployed on any node or namespace independently of where the protected containers might be. The virtual subnet corresponding to the deployment namespace may need have some policies manually configured on Nuage though to allow for traffic to flow from another subnets/namespaces.   
+Nuage does not present a restriction with respect to where the container VNF should be deployed. The VNF can be deployed on any node or namespace independently of where the protected containers might be. The virtual subnet corresponding to the deployment namespace may need have some policies manually configured on Nuage though to allow for traffic to flow from another subnets/namespaces.
 
 ### Container VNF Upgrades
-Upgrading a container VNF already deployed is out of scope for this release.  
+Upgrading a container VNF already deployed is out of scope for this release.
 
 ## Design Changes
-The implementation of this feature will require changes on the OSC entities and operations involved on the definition and registration of VNF images; deployment specification; and monitoring of container VNFs.  
+The implementation of this feature will require changes on the OSC entities and operations involved on the definition and registration of VNF images; deployment specification; and monitoring of container VNFs.
 
 ### Container VNF Images
 In order for a user to deploy a VM VNF using OSC they must register the image file of the VNF along with some additional metadata. For container VNFs this model will be further simplified.  The image for the container is not necessary as it will be retrieved by the virtualization platform from a Docker registry, the user will simply need to provide the metadata needed for OSC to match the VNF with the supported security manager and virtualization platform versions and create the [K8s Deployment object](#kubernetes-deployment).
-This new model is implemented by a couple of new **POST** REST APIs to create the existing types `ApplianceDTO` and `ApplianceSoftwareVersionDTO`. The choice of using two APIs instead of a single one to create both objects is consistent with the existing APIs to GET and DELETE these resources, this also allows the clients to manage appliances and their software versions independently.  
+This new model is implemented by a couple of new **POST** REST APIs to create the existing types `ApplianceDTO` and `ApplianceSoftwareVersionDTO`. The choice of using two APIs instead of a single one to create both objects is consistent with the existing APIs to GET and DELETE these resources, this also allows the clients to manage appliances and their software versions independently.

```

Comment:
Nitpick: Break up the content into logical paragraphs?
add new line after the Deployment object link etc